torchchat/cli/download.py

```diff
@@ -18,15 +18,30 @@
     resolve_model_config,
 )
 
+# By default, download models from HuggingFace to the Hugginface hub directory.
+# Both $HF_HOME and $HUGGINGFACE_HUB_CACHE are valid environment variables for the same directory.
+HUGGINGFACE_HOME_PATH = Path(os.environ.get("HF_HOME", os.environ.get("HUGGINGFACE_HUB_CACHE", os.path.expanduser("~/.cache/huggingface/hub"))))
+
+if os.environ.get("HF_HUB_ENABLE_HF_TRANSFER", None) is None:
+    os.environ["HF_HUB_ENABLE_HF_TRANSFER"] = "1"
+
+# For each model with huggingface distribution path, clean up the old location. 
+def _delete_old_hf_models(models_dir: Path):
```

Comment:
Let's name this more concretely and give more context in the docstring

`delete_old_hf_models` is ambiguous